include/chaiscript/dispatchkit/function_params.hpp

```diff
@@ -36,7 +36,7 @@ namespace chaiscript {
 
       template<size_t Size>
       constexpr explicit Function_Params(const std::array<Boxed_Value, Size> &a)
-        : m_begin(std::begin(a)), m_end(std::end(a))
+        : m_begin(&*std::begin(a)), m_end(&*std::end(a))
```

Comment:
This is not a good fix (not with the STL in VS2019). I had also "fixed" it this way locally at first, but got assertions/crash, e.g. when trying to execute `print("${5}")`, because `std::end(a)` -- with VS STL -- is asserting that it's trying to access with an index that's equal to the array size.

My solution, which works here, is similar to the vector case above: do the addition ourselves:

`m_begin(&a.front()), m_end(&a.front() + Size)`